src/geo_div/geo_div.cpp

```diff
@@ -50,6 +70,11 @@ Polygon_with_holes GeoDiv::largest_polygon_with_holes() const
   return largest_pwh;
 }
 
+const std::vector<Ellipse> GeoDiv::min_ellipses() const
```

Comment:
Clang-Tidy: Return type 'const std::vector<Ellipse>' is 'const'-qualified at the top level, which may reduce code readability without improving const correctness